Aptoide/aptoide-client-v8
# [feature/MOB-753] Login UI Fixes

**What does this PR do?**

   Fixes several UI bugs (and revalidated with Luis) with the login/send magic link layouts.

**Database changed?**

   No

**How should this be manually tested?**

  Check if everything described in the ticket is fixed. Additionally just check if the UI is ok (although it was validated with Luis).

**What are the relevant tickets?**

  [MOB-753](https://aptoide.atlassian.net/browse/MOB-753)

**Code Review Checklist**
